### PR TITLE
Fix primitive declaration double quote bug

### LINKED ISSUE #989: Avro Desrialization Error when Schema is primitive type.
Description
===========

When attempting to use AvroDeserializer with a schema that is a primitive type for example "string" a value deserialization message is received.

Versions:
```
>>> confluent_kafka.version()
('1.5.0', 17104896)
>>> confluent_kafka.libversion()
('1.5.0', 17105151)
```
Confluent Platform: 6.0.0
MacOS

How to reproduce
================
Here is the producer code
``` python
from confluent_kafka.schema_registry import SchemaRegistryClient
from confluent_kafka.schema_registry.avro import AvroSerializer
from confluent_kafka.serializing_producer import SerializingProducer


def delivery_report(err, msg):
    if err is not None:
        print("Delivery failed for User record {}: {}".format(msg.key(), err))
        return
    print(
        "User record {} successfully produced to {} [{}] at offset {}".format(
            msg.key(), msg.topic(), msg.partition(), msg.offset()
        )
    )


# The AVRO Schema
schema_str = '{"type": "string"}'
value = "test"
# Schema Registry Client Initiliazation
schema_registry_configuration = {"url": "http://127.0.0.1:8081"}
schema_registry_client = SchemaRegistryClient(schema_registry_configuration)
# Producer Client Initiliazation
avro_serializer_configutation = {"auto.register.schemas": True}
producer_client_configuration = {
    "bootstrap.servers": "localhost:9092",
    "value.serializer": AvroSerializer(
        schema_str=schema_str, schema_registry_client=schema_registry_client, conf=avro_serializer_configutation
    ),
}
producer = SerializingProducer(producer_client_configuration)
# Produce a record with appropriate headers
producer.produce(topic="test", value=value, on_delivery=delivery_report)
# Flush producer
producer.flush()

```

Here is the consumer code:
```python
from confluent_kafka.schema_registry import SchemaRegistryClient
from confluent_kafka.schema_registry.avro import AvroDeserializer
from confluent_kafka.deserializing_consumer import DeserializingConsumer

# The AVRO Schema
schema_str = '{"type": "string"}'

schema_registry_configuration = {
    "url": "http://localhost:8081",
}
schema_registry_client = SchemaRegistryClient(schema_registry_configuration)
serializer = AvroDeserializer(schema_str=schema_str, schema_registry_client=schema_registry_client)
conf = {
    "client.id": "test_3",
    "bootstrap.servers": "localhost:9092",
    "auto.offset.reset": "earliest",
    "value.deserializer": serializer,  # Serializer used for message values.
    "group.id": "test-group",
}
consumer = DeserializingConsumer(conf)
try:
    consumer.subscribe(["test"])
    while True:
        message = consumer.poll(timeout=5.0)
        # id created to track logic through logs
        if message is None:
            continue
        else:
            print(message.value())
        consumer.commit(asynchronous=True)
except Exception as e:
    raise (e)
    print(e)

```

Running 
`python3 producer.py`
Then:
`python3 consumer.py`
We retrieve the following error from the consumer

```
Traceback (most recent call last):
  File "/Users/.../.pyenv/versions/3.8.1/lib/python3.8/site-packages/confluent_kafka/deserializing_consumer.py", line 137, in poll
    value = self._value_deserializer(value, ctx)
  File "/Users/.../.pyenv/versions/3.8.1/lib/python3.8/site-packages/confluent_kafka/schema_registry/avro.py", line 317, in __call__
    writer_schema = parse_schema(loads(
  File "/Users/.../.pyenv/versions/3.8.1/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/Users/.../.pyenv/versions/3.8.1/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Users/.../.pyenv/versions/3.8.1/lib/python3.8/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting ',' delimiter: line 1 column 11 (char 10)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "consumer.py", line 56, in <module>
    raise (e)
  File "consumer.py", line 39, in <module>
    message = consumer.poll(timeout=5.0)
  File "/Users/.../.pyenv/versions/3.8.1/lib/python3.8/site-packages/confluent_kafka/deserializing_consumer.py", line 139, in poll
    raise ValueDeserializationError(exception=se, kafka_message=msg)
confluent_kafka.error.ValueDeserializationError: KafkaError{code=_VALUE_DESERIALIZATION,val=-159,str="Expecting ',' delimiter: line 1 column 11 (char 10)"}
```
Checklist
=========
Please provide the following information:

 - [x] confluent-kafka-python and librdkafka version (`confluent_kafka.version()` and `confluent_kafka.libversion()`):
 - [x] Apache Kafka broker version:
 - [x] Client configuration: `{...}`
 - [x] Operating system:
 - [x] Provide client logs (with `'debug': '..'` as necessary)
 - [ ] Provide broker log excerpts
 - [ ] Critical issue

### PR DESCRIPTION
**Pull request by dylrich**
This PR fixes #989. We ran into this when we swapped to the new Deserializing Consumer/Producer API and have been monkey patching around it. I added a test case, but am not sure if it's the proper location.

Thanks!

### CODE REVIEW

**Comment by dylrich** [comment]
[clabot:check]

File: tests/schema_registry/test_avro_serializer.py [hunk 1]
```diff
@@ -204,3 +204,16 @@ def test_avro_serializer_subject_name_strategy_default(load_avsc):
...
+def test_avro_serializer_schema_loads_canonical(load_avsc):
+    """
+    Ensures canonical form declarations are correctly parsed by _schema_loads
+    """
+    conf = {'url': TEST_URL}
+    test_client = SchemaRegistryClient(conf)
+    test_serializer = AvroSerializer(test_client,
+                                     load_avsc('canonical_string.avsc'))
+
+    test_serializer._schema.schema_str = '{"type":"string"}'
```
Comment (edenhill) [inline tests/schema_registry/test_avro_serializer.py:218]: What is the purpose of this assignment?

Comment (dylrich) [reply tests/schema_registry/test_avro_serializer.py]: My apologies, I had a typo in that test. I just pushed a change to make that the assert it should have been

**Comment by dylrich** [comment]
I am going to close this in favor of a different PR with more fixes

**Pull request closed without merging**